app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryTask.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import TasksList from '../task/TasksList';
+import PropTypes from 'prop-types';
+
+class ExpandedStoryTask extends Component {
+  constructor(props) {
+    super(props);
```

Comment:
Break the lines between `super`, `this.state` and the other two functions to better readability.
```suggestion
    super(props);
    
    this.state = {}
    
    this.onInput ...
```